src/cloudStorageProvider.js

```diff
@@ -385,24 +385,27 @@ class CloudStorageProvider {
         this.api = new Api(this.context);
 
         if (clearProjects) {
-            this.projects = [];
+            projects = [];
         } else {
-            this.projects.forEach(p => { delete p.files; delete p.branches; delete p.branchId } );
+            projects.forEach(p => { delete p.files; delete p.branches; delete p.branchId } );
         }
     }
 
     refreshUri(uri) {
+        console.log('refreshUri ' + uri.path);
```

Comment:
Just checking that you intended to leave these `console.log`s in the PR...

Reply:
this is for debugging, other extensions usually log stuff. It could help us to debug customer issues.